pom.xml

```diff
@@ -0,0 +1,209 @@
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <!-- Project information -->
+    <name>Global Tracer resolution.</name>
+    <description>Library to obtain a Tracer reference from anywhere in your application.</description>
+    <url>https://github.com/opentracing-contrib/java-globaltracer</url>
+    <inceptionYear>2016</inceptionYear>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <name>Sjoerd Talsma</name>
+            <email>sjoerd@talsma-ict.nl</email>
+            <url>https://github.com/sjoerdtalsma</url>
+            <organization>Talsma ICT</organization>
+        </developer>
+    </developers>
+
+    <scm>
+        <url>https://github.com/opentracing-contrib/java-globaltracer/tree/master</url>
+        <connection>scm:git:git://github.com/opentracing-contrib/java-globaltracer.git</connection>
+        <developerConnection>scm:git:git@github.com:opentracing-contrib/java-globaltracer.git</developerConnection>
+    </scm>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <build.java.version>1.6</build.java.version>
+        <opentracing-api.version>0.20.2</opentracing-api.version>
+    </properties>
+
+    <dependencies>
+        <!-- Runtime dependencies: -->
+        <dependency>
+            <groupId>io.opentracing</groupId>
+            <artifactId>opentracing-api</artifactId>
+            <version>${opentracing-api.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.opentracing</groupId>
+            <artifactId>opentracing-noop</artifactId>
+            <version>${opentracing-api.version}</version>
+        </dependency>
+
+        <!-- Test dependencies: -->
+        <dependency>
+            <groupId>io.opentracing</groupId>
+            <artifactId>opentracing-mock</artifactId>
+            <version>${opentracing-api.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-library</artifactId>
+            <version>1.3</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>1.10.19</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
```

Comment:
same comment about re-using the existing maven infra (as it lets others release for you w/o knowing anything or being dependent on gpg, laptop config etc)